### PR TITLE
PITEAM-151: Replace coreydaley with sayan-biswas

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by coreydaley** [comment]
/assign @adambkaplan 
for lgtm
/assign @sayan-biswas
for review

File: core-services/prow/02_config/openshift/builder/_pluginconfig.yaml [hunk 1]
```diff
@@ -2,13 +2,13 @@ label:
   restricted_labels:
     openshift/builder:
     - allowed_users:
-      - coreydaley
+      - sayan-biswas
       - divyansh42
       - jkhelil
       - otaviof
```
Comment (adambkaplan) [inline core-services/prow/02_config/openshift/builder/_pluginconfig.yaml:8]: filing a TODO to drop Otavio

File: core-services/prow/02_config/openshift/cluster-openshift-controller-manager-operator/_pluginconfig.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ label:
     openshift/cluster-openshift-controller-manager-operator:
     - allowed_users:
       - akram
-      - coreydaley
+      - sayan-biswas
       - ingvagabund
       - soltysh
       - sttts
```
Comment (adambkaplan) [inline core-services/prow/02_config/openshift/cluster-openshift-controller-manager-operator/_pluginconfig.yaml:9]: @jupierce fyi Stefan is no longer with Red Hat.

**Review by adambkaplan — APPROVED** [review]
/approve

/lgtm

**Comment by coreydaley** [comment]
/assign @deads2k 
for approval

**Comment by coreydaley** [comment]
/assign @dennisperiquet @dgoodwin @neisw @stbenjam @xueqzhan
for approval

**Pull request merged**